src/app/directives/is-item-fresh.directive.ts

```diff
@@ -0,0 +1,29 @@
+import { Directive, ElementRef, Input, AfterViewInit } from '@angular/core';
+
+@Directive({
+  selector: '[isItemFresh]'
+})
+export class IsItemFreshDirective implements AfterViewInit {
+  private freshCourse = 14;
+
+  @Input() public isItemFresh: Date;
+
+  constructor(private eleRef: ElementRef) {
```

Comment:
Here you could declare Renderer2 from @angular/core to change DOM more safely.  More info here: http://coldfox.ru/article/5ce1e246f7dc2f02ef7ef429/%D0%98%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D0%B5%D0%BC-Renderer2-%D0%B2-Angular

Reply:
Thank you, I implement this to my next commit.